config/configgrpc/wrappedstream_test.go

```diff
@@ -23,19 +23,24 @@ import (
 	"google.golang.org/grpc"
 )
 
+type ctxKey struct{}
+
+var oneCtxKey = ctxKey{}
+var otherCtxKey = ctxKey{}
```

Comment:
Had to make this change, because lint complained about using "string" as context keys.